app/containers/MessageBox/index.js

```diff
@@ -181,14 +190,23 @@ class MessageBox extends Component {
 	// eslint-disable-next-line react/sort-comp
 	debouncedOnChangeText = debounce((text) => {
 		this.setInput(text);
+		const slashCommand = text.match(/^\/([a-z0-9._-]+) (.+)/im);
```

Comment:
Open slash even without entering text.
User should be able to scroll between available slash commands, since he may not know all of them.

Reply:
Done